src/test/java/com/emc/mongoose/storage/driver/pravega/integration/ReadEventsTest.java

```diff
@@ -0,0 +1,108 @@
+package com.emc.mongoose.storage.driver.pravega.integration;
+
+import com.emc.mongoose.base.data.DataInput;
+import com.emc.mongoose.base.item.DataItem;
+import com.emc.mongoose.base.item.DataItemImpl;
+import com.emc.mongoose.base.item.op.OpType;
+import com.emc.mongoose.base.item.op.data.DataOperation;
+import com.emc.mongoose.base.item.op.data.DataOperationImpl;
+import com.emc.mongoose.base.storage.Credential;
+import com.emc.mongoose.base.storage.driver.StorageDriver;
+import com.emc.mongoose.storage.driver.pravega.PravegaStorageDriver;
+import com.github.akurilov.commons.system.SizeInBytes;
+import lombok.val;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.emc.mongoose.base.Constants.MIB;
+import static com.emc.mongoose.base.item.op.OpType.CREATE;
+import static com.emc.mongoose.base.item.op.OpType.READ;
+import static com.emc.mongoose.base.item.op.Operation.SLASH;
+import static com.emc.mongoose.base.item.op.Operation.Status.SUCC;
+import static org.junit.Assert.assertEquals;
+
```

Comment:
add the `@Ignore` annotation, will fail the test otherwise

Reply:
done